walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +277,159 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeIsIgnored() {
```

Comment:
I feel like these test names (including the existing ones that you didn't write) are bad. "optionalAttendeeIsIgnored" doesn't tell me anything unless I understand the innner workings of your class. What does "ignored" mean>

Prefer something like "optionalAttendee_withConflictingEvent_isNotScheduled"

Reply:
Changed my tests to have more descriptive labels.